cli_connection.py

```diff
@@ -0,0 +1,92 @@
+import asyncio
```

Comment:
there already is a CLI: https://github.com/HENNGE/aapns/blob/master/src/aapns/cli.py

Reply:
Yes I know, the new cli tools are temporary, specifically to exercise different levels in the code

Reply:
can this and `cli_pool.py` be removed now (in favor of aapns/cli.py)?

Reply:
Edited and moved to `examples/`, what do you reckon?